### DISCUSSION TITLE
GPXLAB route planner

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#365] [Show and tell]
https://app.gpxlab.net
 
_"Application routing capabilities are elevated through **BRouter** and routing profiles, finely tuned with SRTM data, enabling you to navigate with precision through a range of terrains and environments."_

GPXLAB is not a navigation software but an online editor to design GPX tracks in advance.
GPXLAB is available online and on mobile (Android / iOS). It uses OSM maps and self hosted **Brouter** service.

[BRouter forum announcement](https://groups.google.com/g/osm-android-bikerouting/c/hKqBiiJceTc)

---

They have a **motorcycle** profile.

**Comment by Leeb64** [comment]
But their motorcycle profile still has a lot of room for improvement...

Set the following in the cruiser driver profile:
AV Toll 80%, AV Motorways 60%, the rest with 0%...then the result is roughly the same!

**Comment by devemux86** [comment]
Everyone better collect your comments and also inform their developers.

It could be done directly or in their BRouter forum announcement:
https://groups.google.com/g/osm-android-bikerouting/c/hKqBiiJceTc
